### ISSUE TITLE
Password encryption

### ISSUE THREAD

**Issue by 3Butterfly0** [rahulpandharkar/Password_Manager#2]
The current implementation of the password manager saves user passwords without robust encryption or uses outdated encryption methods. This exposes a vulnerability where, if the password storage (database or file system) is compromised, the user passwords can be read or extracted in plain text. To enhance security, passwords must be encrypted using modern cryptographic standards.

**Comment by rahulpandharkar** [comment]
Yes, as this project was built as an amateur developer, encryption was not in plan. 
Would be added soon!
